src/lib.rs

```diff
@@ -130,6 +132,14 @@ impl Fairing for RocketSentry {
 
     async fn on_ignite(&self, rocket: Rocket<Build>) -> fairing::Result {
         let figment = rocket.figment();
+        let profile_name = figment.profile().to_string();
+
+        // Set Sentry's environment based on Rocket profile
+        let environment: &'static str = match profile_name {
+            debug if debug == "debug" => "development",
+            release if release == "release" => "production",
+            other => other.leak(),
```

Comment:
It's not nice to leak memory if it can be avoided.

When you have a mix of borrowed (`&'static`) and owned (copied by `to_string()`) data, you can use `Cow::Borrowed`/`Cow::Owned` respectively to integrate them.

The value ends up being wrapped into `Cow` later anyway.